scripts/install.sh

```diff
@@ -69,7 +82,7 @@ printf "Network: $NETWORK\n"
 
 cd $KEEP_BEACON_SOL_PATH
```

Comment:
We should also deploy `threshold-network/solidity-contracts` and link it where needed.

Reply:
Done in [bf09908](https://github.com/keep-network/keep-core/pull/3096/commits/bf09908035cf4e4c237ba4b4d91ce75a28f40fc2)